README.md

```diff
@@ -1 +1,45 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## ABOUT ME
+
+![Kagwiria Nkonge.](https://github.com/Nkonge/hw01/blob/master/Images/Profile_pic.jpg)
+
+<center>
```

Comment:
The centering does not show up in the final document. Markdown is relatively simple in its syntax and capabilities. So while it makes it easier to learn than HTML or LaTeX, it does not allow the same customizability